### ISSUE TITLE
Storyboard shows white

### ISSUE THREAD

**Issue by raresn** [distriqt/AIR-ImageScripts#13]
The storyboard shows white. we used your images to test, with the generate.sh. 
Tested on ios 17 (ipad)

(we had the same problem generating with xcode, so we tried the command line (which was a lot easier btw))
We still add the default imgs as png in the compile command line, not sure if this is relevant.

[LaunchScreen.storyboardc.zip](https://github.com/distriqt/AIR-ImageScripts/files/12660656/LaunchScreen.storyboardc.zip)
[Assets.zip](https://github.com/distriqt/AIR-ImageScripts/files/12660661/Assets.zip)

**Comment by marchbold** [comment]
Have you tried other versions of iOS? We haven't done any testing on iOS 17 yet.

**Comment by raresn** [comment]
Yes, ios 12 and 16 work fine.

**Comment by marchbold** [comment]
Do you have a device with the same screen resolution as the ipad with ios 17 it's failing on?

**Comment by raresn** [comment]
@marchbold yes - we got an ipad with ios 17 at the office and can run whatever test you need :)

**Comment by marchbold** [comment]
Ah, I meant have you tried a device running iOS 16 with this same resolution as the iOS 17 iPad that its' failing on? Just wondering if it's a missing asset from the catalogue or something new in iOS 17

**Comment by raresn** [comment]
iPad pro 11 inch 2nd generation is the model
